packages/suite/src/utils/wallet/accountUtils.ts

```diff
@@ -292,6 +307,48 @@ export const enhanceTokens = (tokens: Account['tokens']) => {
         }));
 };
 
+export const enhanceAddresses = (
```

Comment:
copy pasting comment that I'll include. does it make sense to you?
```
// Since cardano account public key doesn't encode information about the account index
    // Blockfrost backend returns 'i' in address derivation path instead
    // and will rely on the client to replace it with correct account index
```

Reply:
please expand